x-pack/platform/plugins/shared/cases/public/common/use_cases_features.test.tsx

```diff
@@ -48,6 +48,41 @@ describe('useCasesFeatures', () => {
         pushToServiceAuthorized: false,
         observablesAuthorized: false,
         isObservablesFeatureEnabled: true,
+        isExtractObservablesEnabled: false,
+        connectorsAuthorized: false,
+      });
+    }
+  );
+
+  // isObservablesFeatureEnabled, isExtractObservablesEnabled, observables
+  const observableTests: Array<[boolean, boolean, CasesContextFeatures['observables']]> = [
+    [true, true, { enabled: true, autoExtract: true }],
+    [true, false, { enabled: true, autoExtract: false }],
+    [false, false, { enabled: false, autoExtract: true }],
```

Comment:
```suggestion
    [false, true, { enabled: false, autoExtract: true }],
```

Reply:
Oh ignore me, I was reading this line
`returns isObservablesFeatureEnabled=%s and isExtractObservablesEnabled=%s`
as
`WHEN isObservablesFeatureEnabled=%s and isExtractObservablesEnabled=%s`. 

The logic is right, I was just thinking about the literal test title. 👍🏾